scripts/build-android.sh

```
@@ -0,0 +1,238 @@
+    generate_icons "cwebp"
+elif command -v ffmpeg &>/dev/null; then
+    echo "Using ffmpeg"
+    generate_icons "ffmpeg"
+else
+    echo "WARNING: No image tools found (install imagemagick, ffmpeg, or cwebp)"
+    echo "  Skipping icon generation. Install with:"
+    echo "    sudo apt install imagemagick   # or"
+    echo "    sudo apt install ffmpeg"
+fi
+        echo "  ${density}: ${size}x${size}px"
+    done
```

Shellcheck (0.11.0)

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)